data/phi0631/phi001/__cts__.xml

```diff
@@ -11,6 +11,6 @@
   </ti:translation>
 <ti:edition urn="urn:cts:latinLit:phi0631.phi001.perseus-lat4" workUrn="urn:cts:latinLit:phi0631.phi001">
     <ti:label xml:lang="lat">Catilinae Coniuratio</ti:label>
-    <ti:description xml:lang="eng">Sallust</ti:description>
+  <ti:description xml:lang="eng">Sallust. Catilinae Coniuratio.</ti:description>
```

Comment:
Should we start to add **(No bibliographic information found).** or something like that to these files? Just so people don't ask?

Reply:
I think that is  a good plan, until we replace them!